ONSdigital/census-rm-toolbox
# Spike multiple SQL instances

# Motivation and Context
Large Action Rules were affecting the DB performance - making it non-responsive - which would affect time-critical message processing.

# What has changed
Added config for Action DB. Modified DB monitor script to use Action DB. Added scripts for connecting to Action DB.

# How to test?
Terraform a new environment (prod-like) using this branch. Use the kubernates branch of the same name.

Use `dosqla` to connect to Action DB. Use `dosql-i-know-what-i-am-doing-action` to connect in read-write mode. Check the print batch script.

# Links
Trello: https://trello.com/c/fAmdlRkN
